src/oom/oomd-manager.c

```diff
@@ -120,6 +122,7 @@ static int process_managed_oom_message(Manager *m, uid_t uid, sd_json_variant *p
                 if (r < 0 && r != -EEXIST)
                         log_debug_errno(r, "Failed to insert message, ignoring: %m");
 
+                // TODO: Given that these values are updated dynamically on every invocation, it is okay not to handle this as part of reload.
```

Comment:
yeah I think it's fine to leave as-is, they'll be updated on the next round anyway, no different from a restart